test/viewmodels/SecondaryWarehouseDetailsSummarySpec.scala

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Please put these in package `views.detailsPages` (and similar ones below)

Reply:
done